configs/captioning/coco/butd.yml

```diff
@@ -0,0 +1,64 @@
+includes:
+- common/defaults/configs/tasks/captioning/coco.yml
+model_attributes:
+  butd: &butd
+    model_data_dir: ../data/
+    metrics:
+    - type: caption_bleu4
+    losses:
+    - type: caption_cross_entropy
+    classifier:
+      type: language_decoder
+      params:
+        dropout: 0.5
+        hidden_dim: 1024
+        feature_dim: 2048
+        fc_bias_init: 0
+    image_feature_embeddings:
+    - modal_combine:
+        type: top_down_attention_lstm
+        params:
+          dropout: 0.5
+          hidden_dim: 1024
+          attention_dim: 1024
+      normalization: softmax
+      transform:
+        type: linear
+        params:
+          out_dim: 1
+    image_feature_dim: 2048
+    embedding_dim: 300
+    image_feature_encodings:
+    - type: finetune_faster_rcnn_fpn_fc7
+      params:
+        bias_file: detectron/fc6/fc7_b.pkl
+        weights_file: detectron/fc6/fc7_w.pkl
+    inference:
+      type: greedy
```

Comment:
Should we make it beam_search by default? If not, we should add some documentation for this

Reply:
Currently the beam search works only with batch size 1. So adding it as the default inference type is not feasible during training for now. I will create a separate config file that can run beam search inference along with some comments.